doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support intra-pod communications such as between all containers within a pod on localhost |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.09` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) CNF/VNF workloads. |
+| `req.inf.ntw.10` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.11` | Network | The SDN solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.12` | Network | The SDN solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.13` | Network | The Architecture **must** support dual stack IPv4 and IPv6 in Pods and Services. |
+| `req.inf.ntw.14` | Network | The Architecture **must** support Conatiner Network Interfgace (CNI). |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+| req.inf.vir.01   | Virtualisation |   The Architecture **must** support the capability for Containers to consume virtualised compute, storage and network resources.|
+| req.inf.phy.01  |  Physical Infrastructure |   The Architecture **must** support the capability for Containers to consume physical (aka bare metal) compute, storage and network resources. |
 
+<p align="center"><b>Table 2-2:</b> Containerised Platform: Infrastructure Requirements.</p>
+Please note that "shared" is a reference to multi-tenant support and pooled storage resources.
+
+<a name="2.3.3"></a>
+### 2.3.3 Containerised Infrastructure Management (CIM)
```

Comment:
Suggest this is changed to ### 2.3.3 Kubernetes Cluster

Reply:
Thanks - change made